include/rpm/rpmpgp.h

```diff
@@ -33,7 +33,13 @@ typedef struct pgpDig_s * pgpDig;
  */
 typedef struct pgpDigParams_s * pgpDigParams;
 
-typedef uint8_t pgpKeyID_t[8];
+
+/** \ingroup rpmpgp
+ * The length (in bytes) of a binary (not hex encoded) key ID.
+ */
+#define PGP_KEYID_LEN 8
```

Comment:
Is there any chance that this could be changed to be a fingerprint in the future?  Yes, I know that is a separate issue :)

Reply:
It's not possible to replace the use of Iey IDs with fingerprint without reworking the callers.  First, the fingerprint is not always available.  For instance, many signatures only include an issuer subpacket with a Key ID and not an issuer fingerprint subpacket with a fingerprint.  Second, v3 and the upcoming v5 fingerprints are different sizes from v4 fingerprints.  As such, a fixed sized buffer, which rpm currently uses, is not reasonable.